includes/reader-revenue/class-reader-revenue-emails.php

```diff
@@ -0,0 +1,422 @@
+			]
+		);
+	}
+
+	/**
+	 * Send an HTML email.
+	 *
+	 * @param string|int $type_or_post_id Email type or email post ID.
+	 * @param string     $to Recipient's email addesss.
+	 * @param array      $placeholders Dynamic content substitutions.
+	 */
+	public static function send_email( $type_or_post_id, $to, $placeholders = [] ) {
+		if ( ! self::supports_emails() ) {
+			return false;
+		}
+		if ( 'string' === gettype( $type_or_post_id ) ) {
+			$email_config = self::get_email_config_by_type( $type_or_post_id );
+		} elseif ( 'integer' === gettype( $type_or_post_id ) ) {
+			$email_config = self::serialize_email( null, $type_or_post_id );
+		} else {
+			return false;
+		}
+		if ( ! $to || false === $email_config || 'publish' !== $email_config['status'] ) {
+			return false;
+		}
+		$html           = $email_config['html_payload'];
+		$from_email     = $email_config['from_email'];
+		$placeholders[] = [
+			'template' => self::DYNAMIC_CONTENT_PLACEHOLDERS['CONTACT_EMAIL'],
+			'value'    => sprintf( '<a href="mailto:%s">%s</a>', $from_email, $from_email ),
+		];
+		foreach ( $placeholders as $value ) {
+			$html = str_replace(
+				$value['template'],
+				$value['value'],
+				$html
+			);
+		}
+		$email_content_type = function() {
+			return 'text/html';
+		};
+		add_filter( 'wp_mail_content_type', $email_content_type );
+		$email_send_result = wp_mail( // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.wp_mail_wp_mail
+			$to,
+			$email_config['subject'],
+			$html,
+			[
+				sprintf( 'From: %s <%s>', $email_config['from_name'], $from_email ),
+			]
+		);
+		remove_filter( 'wp_mail_content_type', $email_content_type );
+		return $email_send_result;
+	}
+
+	/**
+	 * Load a template of an email.
+	 *
+	 * @param string $type Email type.
+	 */
+	private static function load_email_template( $type ) {
+		$templates_directory = dirname( NEWSPACK_PLUGIN_FILE ) . '/includes/templates/reader-revenue-emails/';
+		return require $templates_directory . $type . '.php';
+	}
+
+	/**
+	 * Does this instance support emails?
+	 */
+	public static function supports_emails() {
+		if ( ! class_exists( 'Newspack_Newsletters' ) ) {
+			return false;
+		}
+		return true;
+	}
+
+	/**
+	 * Can email of a particular type be sent?
+	 *
+	 * @param string $type Type of the email.
+	 */
+	public static function can_send_email( $type ) {
+		if ( ! self::supports_emails() ) {
+			return false;
+		}
+		$email_config = self::get_email_config_by_type( $type );
+		if ( ! $email_config || 'publish' !== $email_config['status'] ) {
+			return false;
+		}
+		return true;
+	}
+
+	/**
+	 * Serialize an email config.
+	 *
+	 * @param string $type Type of the email.
+	 * @param int    $post_id Email post id.
+	 */
+	private static function serialize_email( $type = null, $post_id = 0 ) {
+		if ( ! self::supports_emails() ) {
+			return false;
+		}
+		switch ( $type ) {
+			case 'receipt':
+				$label       = __( 'Receipt', 'newspack' );
```

Comment:
Non-blocking: what about calling it a "custom" receipt, since the alternative (if the email isn't active) is the default Stripe receipt?

```suggestion
				$label       = __( 'Custom Receipt', 'newspack' );
```